src/project/manifest/pyproject.rs

```diff
@@ -73,6 +73,18 @@ impl From<PyProjectManifest> for ProjectManifest {
             }
         }
 
+        // Add the project itself as an editable dependency
+        if let Some(name) = &item.project.as_ref().map(|p| &p.name) {
```

Comment:
No sure what we should do if the project.name is not define. Maybe display a warning?

Reply:
Yeah we kind of need the name, to know what to name the package in the `site-packages` folder. As it is required by this struct

```rust
#[derive(Debug, Clone, Eq, PartialEq)]
pub enum RequirementOrEditable {
    Editable(PackageName, requirements_txt::EditableRequirement),
    Pep508Requirement(pep508_rs::Requirement),
}
```

Reply:
indeed. I'll add a warning saying the project has not been installed in an else branch

Reply:
Actually, name has to be defined in the schema (if project is).